### ISSUE TITLE
Numap keys don't work on El Capitan

### ISSUE THREAD

**Issue by alexandrudima** [Microsoft/vscode#6833]
From https://github.com/Microsoft/vscode/issues/983#issuecomment-221346786
From @cliffowen 

Again, I am using a Mac.. and using El Capitan (since I did not mention that before). The keyboard layout is ANSI US. I speculate the layout is correct though, as VMWare and Parallels both handle the keyboard properly. Actual keyboard is a Logitech Wave.

Oddly enough, Numpad + and - are fine (numpad_add and numpad_subtract). It's the numbers that are not correct. In the capture window, they report 0, 1, 2... rather than numpad0, numpad1, etc.

Here are a couple of the bindings I had set up and were working on the Mac on 1.0 that no longer work with 1.1.x. My primary goal is identical behavior between VS and VSC on PC and Mac.

```
{
    "key": "numpad9",
    "command": "cursorPageUp",
    "when": "editorTextFocus"
},
{
    "key": "shift+numpad8",
    "command": "cursorUpSelect",
    "when": "editorTextFocus"
},
```

**Comment by alexandrudima** [comment]
@cliffowen Now that I think about it, maybe you need to press `NumLock`? `NumLock` toggles the numpad9 to be page up.

@aeschli Can you please help me out and confirm this is indeed a bug -- you use a mac with an external keyboard I think?

**Comment by aeschli** [comment]
@weinand Andre has a external keyboard with numblock.

**Comment by cliffowen** [comment]
@alexandrudima Yeah, NumLock is the very first thing to try. But alas, no. Mac doesn't really know about the NumLock. No matter what, numpad9 is just 9 now.

**Comment by alexandrudima** [comment]
I have validated that we now (i.e. after PR #22894 aka #17521 -- in today's Insiders build) appear to behave correctly for numpad keys. I have attached a standard QWERTY keyboard to a mac mini and configured OSX to use the US layout. Pressing numpad keys appear to be interpreted correctly:

![image](https://cloud.githubusercontent.com/assets/5047891/24293291/5668a700-1091-11e7-8d03-6c5b96dfc87d.png)
